tools/Makefile.unix

```diff
@@ -381,9 +370,9 @@ include tools/LibTargets.mk
 # is an archive.  Exactly what is performed during pass1 or what it generates
 # is unknown to this makefile unless CONFIG_PASS1_OBJECT is defined.
 
-pass1: $(USERLIBS)
+pass1: tools/incdir$(HOSTEXEEXT) $(USERLIBS)
```

Comment:
This change will break the concurrent build since tools/incdir and library in $(USERLIBS) and $(NUTTxLIBS) build at the same time.

Reply:
I was concerned about this part as well, but it looks like the nightly build went well for today.

Reply:
This is a race condition doesn't always happen, just like the mulitple thread issue.

Reply:
Yes, I understand that.
What do you think about what's described here: https://stackoverflow.com/questions/16829933/how-can-i-use-notparallel-in-makefile-only-on-specific-targets 
In our example it could be something like:
```Make
pass1libs: $(USERLIBS)
pass1:
	$(MAKE) tools/incdir$(HOSTEXEEXT)
	$(MAKE) pass1libs

pass2libs: $(NUTTXLIBS)
pass2: 
	$(MAKE) tools/incdir$(HOSTEXEEXT)
	$(MAKE) pass2libs
```
Or even
```Make
pass1libs: $(USERLIBS)
pass1: tools/incdir$(HOSTEXEEXT)
	$(MAKE) pass1libs

pass2libs: $(NUTTXLIBS)
pass2: tools/incdir$(HOSTEXEEXT)
	$(MAKE) pass2libs
```
I guess also the order-only dependency answer could work (if acceptable in our Makefiles):

```Make
pass1: $(USERLIBS) | tools/incdir$(HOSTEXEEXT)

pass2: $(NUTTXLIBS) | tools/incdir$(HOSTEXEEXT)
```
This would insure that the libs are not built before incdir (but will not update pass1/2 if incdir is changed)

Reply:
#1152 removes this dependency, so nothing will be needed.